File: 8kyu/count-by-x/count-by-x.go
package main

import "fmt"

func CountBy(x, n int) []int {
  res := make([]int, 0)

	for i := 1; i <= n; i += 1 {
		res = append(res, x * i)
	}

	return res
}


func main() {
	fmt.Println(CountBy(1,5))
	fmt.Println(CountBy(2,5))
}

File: 8kyu/count-by-x/count-by-x.py
def count_by(x, n):
  """
  Return a sequence of numbers counting by `x` `n` times.
  """
  res = []
  for num in range(1, n + 1):
    res.append(num * x)
  return res


print(count_by(1, 5))
print(count_by(2, 5))